packages/fx-core/resource/package.nls.json

```diff
@@ -287,8 +289,8 @@
   "core.createProjectQuestion.projectType.outlookAddin.title": "App Features Using an Outlook Add-in",
   "core.createProjectQuestion.projectType.tab.detail": "Embed your own web content in Teams, Outlook, and the Microsoft 365 app",
   "core.createProjectQuestion.projectType.tab.title": "App Features Using a Tab",
-  "core.createProjectQuestion.projectType.copilotPlugin.detail": "Create a plugin to extend Microsoft 365 Copilot using your APIs",
-  "core.createProjectQuestion.projectType.copilotPlugin.label": "Plugin",
+  "core.createProjectQuestion.projectType.copilotPlugin.detail": "Create a plugin to extend Microsoft Copilot for Microsoft 365 using your APIs",
+  "core.createProjectQuestion.projectType.copilotPlugin.label": "Plugin for Copilot",
```

Comment:
Suggestion: Copilot Plugin
Logic: Easier to perceive

Reply:
@MuyangAmigo what's your opinion on this? I thought we have Figma designed as "Copilot plugin". Should we update as what Surpiya suggested and update the Figma?

Reply:
OK we could chang to Copilot Plugin.

Reply:
@supkasar updated as suggested.